examples/example1.json

```diff
@@ -0,0 +1,27 @@
+{
+	"id": "31e9d66a-cd83-4174-9429-b932f1abe1be",
+	"version": "v04",
+	"type": "Feature",
+	"geometry": {
+		"type": "Point",
+		"coordinates": [
+			6.146255135536194,
+			46.223296618227444
+		]
+	},
+	"properties": {
+		"pubtime": "2022-03-20T04:50:18.314854383Z",
+		"datetime": "2022-03-20T04:45:00Z",
+		"integrity": {
+			"method": "sha512",
+			"value": "A2KNxvks...S8qfSCw=="
+		},
+		"data_id": "dataset/123/data-granule/UANT01_CWAO_200445___15103.bufr4",
```

Comment:
så dette er en melding om en bufr-fil? Slik jeg forstår det nå, så skal bufr-dataene leses ut, berikes med metadata (f.eks. posisjon) og skrives til json som deretter sendes i køen slik som det andre eksempelet over. I tillegg skal datene legges i "Data and Metadata Store", men jeg går ut fra at de kan bruke MQTT-meldingen som input iom at den skal være komplett med data og metadata. Uansett fint å ha noen eksempel BUFR-filer, kode til å konvertere til MQTT og selve MQTT-filene.